app/routes/home.js

```diff
@@ -23,7 +23,9 @@ function showError(){
 export default Ember.Route.extend({
     setupController: function(controller, model) {
         controller.set('model' , model);
-
+        document.addEventListener("dragend", ()=>{
+           controller.set('isDragging', false)
+       })
```

Comment:
We'll need to find a better place to put this, and also remove the event listener when they leave the route